[Fix] allow enable_hpi=1 for cyrillic_PP-OCRv5_mobile_rec on GPU

alealexpro100 opened the pull request:
Export to ONNX for eslav_PP-OCRv5_mobile_rec did not work then option enable_hpi=1 was set (tested on nvidia GPU, paddle 3.2.0).
Similar issue: https://github.com/PaddlePaddle/PaddleOCR/issues/16450

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PaddlePaddle/PaddleX?pullRequest=4860) <br/>All committers have signed the CLA.

Bobholamovic commented:
Please sign the CLA and merge the commits to the `develop` branch.

Bobholamovic commented:
Please also add the model info to the fields other than `paddle311` in the JSON file accordingly.

alealexpro100 committed:
[Fix] Allow HPI for cyrillic_PP-OCRv5_mobile_rec on Paddle 3.11 CUDA 12

This fix allows ONNX export for `cyrillic_PP-OCRv5_mobile_rec` model when used with Paddle 3.11 and CUDA 12.

alealexpro100 commented:
Checked other modes (`cpu_x64`, `gpu_cuda11`, `gpu_cuda12`) and versions (`paddle30`, `paddle31`, `paddle311`) - all of them already have `eslav_PP-OCRv5_mobile_rec` model info. Only `gpu_cuda12` - `paddle311` lacks it.
Also just rebased branch with latest changes from `develop` branch in upstream. Also changed commit message to make it more clear.

mattheliu commented:
@alealexpro100  Thanks for your contribution! You will receive a beautiful PaddlePaddle gift. Please provide your mailing address and phone number by filling out the following questionnaire before January 23th,2026.

Looking forward to the future, we will walk further together in the world of open source!
Click Here ：https://paddle.wjx.cn/vm/wEUMTJJ.aspx#
